pkg/machines/components/diskAdd.jsx

```diff
@@ -433,6 +437,9 @@ export class AddDiskModalBody extends React.Component {
         }
 
         // use existing volume
+        if (!this.state.existingVolumeName)
```

Comment:
Same for the preselected options comment as above.

Reply:
Same, is user has some pool with no volumes, this.state.existingVolumeName is undefined, but we do not prevent them from clicking "Add" and attachDisk action being called.